platform/node/src/node_conversion.cpp

```diff
@@ -5,120 +5,104 @@ namespace mbgl {
 namespace style {
 namespace conversion {
 
-static const v8::Local<v8::Value>& cast(const Storage& storage) {
-    return *static_cast<const v8::Local<v8::Value>*>(static_cast<const void*>(&storage));
-}
-
-static v8::Local<v8::Value>& cast(Storage& storage) {
-    return *static_cast<v8::Local<v8::Value>*>(static_cast<void*>(&storage));
-}
-
-static void destroy(Storage& storage) {
-    const v8::Local<v8::Value> value(std::move(cast(storage)));
-    (void)value; // appease linter
-}
+using V8Value = v8::Local<v8::Value>;
```

Comment:
Not sure this typedef adds much value. I'd rather just use `v8::Local<v8::Value>` directly.